tests/e2e/assets/neuromation-client/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-slim
+FROM python:3.7-slim
```

Comment:
Is it required? 3.6 is the minimal requirement for the client.

Reply:
Technically 3.6 works as well. For 3.7 the installation is slightly faster because a couple of packages are skipped.